static/js/components/Dashboard.js

```diff
@@ -14,7 +14,7 @@ class Dashboard extends React.Component {
 
   render() {
     const { profile, dashboard, expander, dispatch } = this.props;
-    let imageUrl = (SETTINGS.edx_base_url + '/static/images/profiles/default_120.png').
+    let imageUrl = `${SETTINGS.edx_base_url}/static/images/profiles/default_120.png`.
```

Comment:
It would be nice to use a path join library similar to `os.path.join` in Python, but out of scope for this issue